generate.sh

```diff
@@ -0,0 +1,29 @@
+mkdir temp
+
+#cp rep/doc temp/
+
+cd temp
+sphinx-quickstart -q -v $VERSION -a $AUTHOR -p $PROJECTNAME
+rm index.rst
+cd ..
+cp -a $DOCPATH. temp/
+cd temp
+make html
+git config --global user.email $EMAIL
+git config --global user.name $USERNAME
+
+
+
+git clone --quiet --branch=gh-pages $GITURL gh-pages
+chmod -R u+x gh-pages
```

Comment:
when i tried removing folder without permission , error showing that no access to remove .git folder in gh-pages